index.html

```diff
@@ -291,7 +301,18 @@ <h3>What is a Verifiable Credential?</h3>
         </p>
 
         <p>
-<a>Holders</a> of <a>verifiable credentials</a> can generate
+There are 2 broad categories of <a>verifiable credentials</a>: <a>bound credentials</a> and <a>unbound credentials</a>. 
+<a>Bound credentials</a> are bound a specific credential <a>subject</a>. 
+<a>Unbound credentials</a> are not bound to a <a>subject</a>. 
+As such, <a>unbound credentials</a> cannot be used to create <a>verifiable presentations</a> -
+they are simply "data that has been signed by an <a>Issuer</a> and held by any <a>Holder</a>".
+<a>Unbound credentials</a> are easily issued to and exchanged between groups of two or more <a>Holders</a>. 
+The most common use case for <a>unbound credentials</a> is to represent 
+all types of business documents like invoices and purchase orders 
+(that are not simple extensions or representations of a single subject).
```

Comment:
Hrm, disagree with this language. Business documents, invoices, and purchase orders typically have identifiers associated with them that would be the `credentialSubject` identifier.